source/MissionAction.cpp

```diff
@@ -497,6 +542,14 @@ MissionAction MissionAction::Instantiate(map<string, string> &subs, const System
 	result.gifts = gifts;
 	result.requiredOutfits = requiredOutfits;
 	result.payment = payment + (jumps + 1) * payload * paymentMultiplier;
+	if(fine)
+	{
+		result.fine = fine;
+		subs["<fine>"] = Format::Number(abs(result.fine))
```

Comment:
The code in ::Load, and use of "is positive" instead of just non-zero means we don't need to call `abs` here.

This should also use Format::Credits, not Format::Number.